Minesweeper.py

```diff
@@ -187,21 +211,18 @@ def draw(self):
             self.win.fill(Color.BACKGROUND)
             self.game_won_screen.draw()
 
-        if self.gamestate.game_won:
-            self.game_won_screen.draw()
-
-        if self.gamestate.game_over == True or self.gamestate.game_won == True:
-            self.game_won_screen.handle_events(events)
-
         if self.start_time != 0 and not self.gamestate.game_over and not self.gamestate.game_won:
             self.time_elapsed = (pygame.time.get_ticks() - self.start_time) // 1000
 
-        if self.gamestate.tutorial == False:
+        if not self.gamestate.tutorial:
```

Comment:
The game timer is a separate concern, thus readability would be improved if we would move it (including the time_elapsed and start_time) into a separate class.